tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -1120,8 +1120,10 @@ void test() {
 #if !defined(_DLL) || _ITERATOR_DEBUG_LEVEL == DEFAULT_IDL_SETTING
     test_locale<wchar_t>();
     test_locale<char>();
+#ifndef _MSVC_INTERNAL_TESTING // TRANSITION, the Windows version on Contest VMs doesn't always understand ".UTF-8"

```

Comment:
is there some internal ticket for fixing this we can reference?

Reply:
Good idea. I've filed internal [IcM-349538021](https://portal.microsofticm.com/imp/v3/incidents/details/349538021/home) "Outdated OSes on private Contest runners" to track this.

We haven't cited IcM "incidents" in the source code before; given your approval I will assume that the TRANSITION comments, git/PR history, and the filed incident are sufficient for us to remember what happened here.

Reply:
Do you want it in the MSVC-PR only, or also the GH PR?

Reply:
I'd prefer both.

Reply:
Welp, now that the IcM ticket has been closed and Contest supposedly decommissioned, shall we abandon this PR?

Reply:
It's not totally decommissioned, as apparently some LKG test runs are still running Contest-Private checks, but I'd be happy to abandon this PR. We've finally officially Gotten The Memo about running `runall-private.ps1`, I've verified that it succeeds without workarounds for x86 and x64, and we've clearly survived for months with the status quo. (Reportedly, Phase 6 will truly put an end to Contest.)